### PR TITLE
Basic routing

### LINKED ISSUE #5: Add react-router
- route for newdle creation page (`/new`?)
- default 404 route

### PR DESCRIPTION
**Pull request by mvidalgarcia**
closes #5

### CODE REVIEW

File: newdle/client/src/Newdle.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+export default function Newdle() {
```
Comment (ThiefMaster) [inline newdle/client/src/Newdle.js]: `CreateNewdle` or something like that?

File: newdle/client/src/App.js [hunk 2]
```diff
@@ -1,16 +1,27 @@
...
+    <Router>
+      <main>
+        <header className={styles.header}>
+          <Header as="h1" className={styles.title}>
+            <Link to="/">newdle</Link>
+          </Header>
+        </header>
+        <Switch>
+          <Route exact path="/" component={Home} />
+          <Route path="/new" component={Newdle} />
```
Comment (ThiefMaster) [inline newdle/client/src/App.js]: I think we want `exact` here as well

**Comment by mvidalgarcia** [comment]
Rebased and updated.

File: newdle/client/src/CreateNewdle.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+export default function Newdle() {
```
Comment (ThiefMaster) [inline newdle/client/src/CreateNewdle.js]: `CreateNewdle`

**Pull request merged**